packages/salesforcedx-vscode-core/test/jest/telemetry/index.test.ts

```diff
@@ -95,7 +97,7 @@ describe('Telemetry', () => {
       expect(telemetryEnabled).toEqual(true);
 
       await showTelemetryMessage(mockExtensionContext);
-      expect(globalStateTelemetrySpy).toHaveBeenCalledTimes(2);
+      expect(globalStateTelemetrySpy).toHaveBeenCalledTimes(3);
```

Comment:
Why'd this change?  Should we test for the third value below here like we do for the other 2?

Reply:
The function `UserService.getTelemetryUserId` fetches the global state to get value for `TELEMETRY_GLOBAL_USER_ID`. The bug I introduced prevented the call from happening since extensionContext was always undefined.

Now that the bug is fixed, I needed to adjust this check since there is an additional call to global state `get`.

Reply:
I can never make those code suggestions work.

Can you add the following after the above change so we check for the three expected values? 
```
      expect(globalStateTelemetrySpy).toHaveBeenCalledWith(TELEMETRY_GLOBAL_USER_ID);
```